pkg/controller/networkpolicy/validate.go

```diff
@@ -236,6 +251,22 @@ func (v *NetworkPolicyValidator) Validate(ar *admv1.AdmissionReview) *admv1.Admi
 			}
 		}
 		msg, allowed = v.validateAntreaPolicy(&curANNP, &oldANNP, op, ui)
+	case "AdminNetworkPolicy":
```

Comment:
I don't see a corresponding `ValidatingWebhookConfiguration` in the manifest. Will that code be reachable?

Reply:
Fixed